app/3.0/docs/tools/polymer-cli-commands.md

```diff
@@ -277,9 +233,11 @@ Sets npm mode. Dependencies are installed from npm, the component directory is s
 polymer serve --npm --module-resolution="node"
 ```
 
+Because the dependencies for Polymer 3.0 projects are defined in `package.json`, the `--npm` flag is usually required. You will probably find it easier to [add this option to your polymer.json configuration](polymer-json).
+
 ### --module-resolution
 
-Specifies how to resolve module specifiers in import and export statements when rewriting them to be web-compatible.
+Specifies how to resolve module specifiers in import and export statements when rewriting them to URLs.
 
 Valid values are `"none"` and `"node"`. Defaults to `"none"`. 
```

Comment:
Now defaults to `node`. This isn't a breaking change for 1.x or 2.x projects, so most people should not need to use this flag on any version.